app.rb

```diff
@@ -138,8 +138,9 @@ def config
     def authorization_url(ial:, aal: nil, enable_attempts_api: nil)
       endpoint = openid_configuration[:authorization_endpoint]
       irs_attempts_api_session_id = enable_attempts_api ? random_value : nil
+      issuer = enable_attempts_api ? config.mock_irs_client_id : config.client_id
       request_params = {
-        client_id: config.client_id,
```

Comment:
This is one of several references to `client_id`, would we need to handle each? Notably, the one in `client_assertion_jwt` seems important. When testing this locally, I received a `JWT::DecodeError` when redirected back to the sample app, which seems related.

Reply:
Indeed - thanks! Done in 2227e81

Reply:
Rebased - now done in 9d6bd32